virtual_rainforest/models/animals/constants.py

```diff
@@ -7,7 +7,7 @@
 
 
 @dataclass
-class Term:
+class ScalingTerms:
```

Comment:
If you're using this classes this way, and they are meant to be immutable, I would use [`NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple) rather than a `dataclass`.